modal-handler.js

```
@@ -0,0 +1,37 @@
+function initModal() {
+    const modal = document.querySelector(".modal");
+    const modalText = document.querySelector(".modal-text");
+    const closeBtn = document.querySelector(".close-btn");
+
+    if (!modal || !modalText) return;
```

Instead of checking for null and returning, consider removing this check and allowing the script to throw an error if modal or modalText is not found. This will make debugging easier by providing a clear error message.

---

Done